apps/ui/src/views/Bridge/Ethereum/BridgeOperation/index.tsx

```diff
@@ -312,17 +312,25 @@ export const BridgeOperationForm: React.FC = () => {
               createInstance()
                 .addrs(value)
                 .then((d) => {
-                  const addresses = d.filter(
-                    (addr) =>
-                      addr.key ===
-                      `address.${
-                        direction === BridgeDirection.In
-                          ? 'ckb'
-                          : network === 'Ethereum'
-                          ? 'eth'
-                          : network.toLowerCase()
-                      }`,
-                  );
+                  const addresses = d
+                    .filter(
+                      (addr) =>
+                        addr.key ===
+                          `address.${
+                            direction === BridgeDirection.In
+                              ? 'ckb'
+                              : network === 'Ethereum'
+                              ? 'eth'
+                              : network.toLowerCase()
+                          }` &&
+                        addr.value &&
+                        addr.value.length > 0,
+                    )
+                    .map((addr) =>
+                      addr.key === 'address.ckb'
+                        ? { ...addr, value: (envNetwork === 'mainnet' ? 'ckb' : 'ckt') + addr.value.substring(3) }
```

Comment:
Conversion between mainnet and testnet is not simply replacing the prefix, it should be transformed to script and returned with the correct prefix.

See the case at https://github.com/rev-chaos/ckb-address-demo

> == full address test (hash_type = 0x01) ==
code_hash to encode:     9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8
with args to encode:     b39bbc0b3673c7d36450bc14cfcdad2d559c6c64
mainnet address:	 ckb1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqdnnw7qkdnnclfkg59uzn8umtfd2kwxceqxwquc4
testnet address:	 ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqdnnw7qkdnnclfkg59uzn8umtfd2kwxceqgutnjd

Reply:
Refactored to that, when ckb address prefix is different from env, parse it to lockScript and then encode to the correct address